### PR TITLE
modules/understanding-upgrade-channels: Mention 4.(y-1) in 4.y channels

### PR DESCRIPTION
**Pull request by codyhoag**
Supersedes #31365

Preview: https://deploy-preview-31919--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-between-minor.html#understanding-upgrade-channels_updating-cluster-between-minor

### CODE REVIEW

**Comment by codyhoag** [comment]
I will manually backport these changes to previous versions, since hard-coded versions will need to be updated per release branch.

**Comment by codyhoag** [comment]
@jiajliu PTAL. Thanks!

**Review by wking — APPROVED** [review]
/lgtm

**Comment by jiajliu** [comment]
LGTM, thx.

File: modules/understanding-upgrade-channels.adoc [hunk 1]
```diff
@@ -50,7 +52,7 @@ for more build information.
...
+The `fast-{product-version}` channel is updated with new {product-version} and previous minor versions as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
```
Comment (adellape) [inline modules/understanding-upgrade-channels.adoc:55]: Might be misunderstanding, but the first sentence read a little funny to me. Could it be something like:

```
The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability release.
```

?

Comment (codyhoag) [reply modules/understanding-upgrade-channels.adoc]: Agreed! Just made that update :+1: . Thanks!

**Pull request merged**